src/index.ts

```diff
@@ -91,3 +91,5 @@ export function render(
     })
     .catch((error) => showError(el, error));
 }
+
+export * from "./widget";
```

Comment:
Doesn't this mean we make the main export have the widget export as well. Can you explain why this is needed?

Reply:
I think we need this now since the extension uses the widget.